documentation/modules/ROOT/pages/configuration/storage.adoc

```diff
@@ -0,0 +1,705 @@
+|Default
+|Description
+
+|[[schema-history-internal-kafka]]<<schema-history-internal-kafka,`schema.history.internal`>>
+|No default
+|Must be set to `io.debezium.storage.kafka.history.KafkaSchemaHistory`.
+
+|[[schema-history-internal-kafka-topic]]<<schema-history-internal-kafka-topic,`schema.history.internal.kafka.topic`>>
+|No default
+|The name of the topic for the database schema history
+
+|[[schema-history-internal-kafka-bootstrap-servers]]<<schema-history-internal-kafka-bootstrap-servers,`schema.history.internal.kafka.bootstrap.servers`>>
+|No default
+|A list of host/port pairs that the connector will use for establishing the initial connection to the Kafka cluster for retrieving database schema history previously stored by the connector. This should point to the same Kafka cluster used by the Kafka Connect process.
+
+|[[schema-history-internal-kafka-recovery-poll-interval-ms]]<<schema-history-internal-kafka-recovery-poll-interval-ms,`schema.history.internal.kafka.recovery.poll.interval.ms`>>
+|100
+|The number of milliseconds to wait while polling for persisted data during recovery.
+
+|[[schema-history-internal-kafka-recovery-attempts]]<<schema-history-internal-kafka-recovery-attempts,`schema.history.internal.kafka.recovery.attempts`>>
+|100
+|The number of attempts in a row that no data are returned from Kafka before recover completes. The maximum amount of time to wait after receiving no data is (recovery.attempts) x (recovery.poll.interval.ms).
+
+|[[schema-history-internal-kafka-query-timeout-ms]]<<schema-history-internal-kafka-query-timeout-ms,`schema.history.internal.kafka.query.timeout.ms`>>
+|3
+|The number of milliseconds to wait while fetching cluster information using Kafka admin client.
+
+|[[schema-history-internal-kafka-create-timeout-ms]]<<schema-history-internal-kafka-create-timeout-ms,`schema.history.internal.kafka.create.timeout.ms`>>
+|30
+|The number of milliseconds to wait while create kafka history topic using Kafka admin client.
+
+|[[schema-history-internal-producer]]<<schema-history-internal-producer,`schema.history.internal.producer.*`>>
+|No Default
+|Pass-through properties prefix for configuring how producer client interact with schema history topics.
+
+|[[schema-history-internal-consumer]]<<schema-history-internal-consumer,`schema.history.internal.consumer.*`>>
+|No Default
+|Pass-through properties prefix for configuring how consumer client interact with schema history topics.
+
+|===
+
+
+== File
+
+It's possible to persist the position (offsets) of the connector in a local file on disk. This allows Debezium to resume from where it left off after a restart. It's simple, fast, and ideal for single-node applications or testing scenarios.
+
+=== Offset Store
+
+[cols="35%a,10%a,55%a",options="header"]
+|===
+|Property
+|Default
+|Description
+
+|[[offset-storage-file]]<<offset-storage-file,`offset.storage`>>
+|No default
+|Must be set to `org.apache.kafka.connect.storage.FileOffsetBackingStore`
+
```

Comment:
If this and similar properties have required values, can they be coded to carry the appropriate defaults?